files/lib/data/faq/QuestionAction.class.php

```diff
@@ -113,19 +116,18 @@ public function create()
 
     /**
      * @inheritDoc
-     * https://github.com/WoltLab/WCF/blob/master/wcfsetup/install/files/lib/data/reaction/type/ReactionTypeAction.class.php#L46
+     * Adapted from: https://github.com/WoltLab/WCF/blob/9ed5bd7220ff1beec1949dc13777bf2f62acf1f5/wcfsetup/install/files/lib/data/reaction/type/ReactionTypeAction.class.php#L112
      */
     public function update()
     {
         //check if showOrder must be updated
-        if (\count($this->objects) == 1 && isset($this->parameters['data']['showOrder'])) {
+        if (isset($this->parameters['data']['showOrder']) && \count($this->objects) === 1) {
```

Comment:
Warum die andere Sortierung in dem if?

Reply:
In der Praxis unwesentlich, aber hier geht es mir um das Prinzip: Die Operation `isset` ist günstiger als der Funktionsaufruf von `count`, weshalb man die Bedingungen auch entsprechend so anordnen sollte, um prinzipiell so performanten Code wie möglich zu schreiben.